docs/source/install.ipynb

```diff
@@ -30,9 +29,21 @@
     "```\n",
     "\n",
     "#### Add-ons\n",
+    "You can install add-ons individually or all at once by running:\n",
+    "```bash\n",
+    "pip install evalml[complete]\n",
+    "```\n",
+    "\n",
+    "**Time Series Support** <br>\n",
```

Comment:
I see <br> and \n here but it doesn't show up here?

![image](https://user-images.githubusercontent.com/5422235/129754331-5d1707cd-2749-41d4-a88e-7e2e501259fc.png)

Reply:
Fixed https://github.com/alteryx/evalml/pull/2627/commits/f208e624c4ee487d7b76dca6afef7f4be2496a79